bond/bond.go

```diff
@@ -56,15 +56,24 @@ func loadConfigFile(bytes []byte) (*bondingConfig, string, error) {
 	if err := json.Unmarshal(bytes, bondConf); err != nil {
 		return nil, "", fmt.Errorf("Failed to load configuration file, error = %+v", err)
 	}
+        if bondConf.IPAM.Type == "bond"{
```

Comment:
introduce "bond"  as a constant string at the top of the file instead of typing it out.